packages/plugin-ext/src/main/browser/plugin-ext-frontend-module.ts

```diff
@@ -70,6 +70,12 @@ import { WebviewResourceLoader, WebviewResourceLoaderPath } from '../common/webv
 import { WebviewResourceCache } from './webview/webview-resource-cache';
 import { PluginIconThemeService, PluginIconThemeFactory, PluginIconThemeDefinition, PluginIconTheme } from './plugin-icon-theme-service';
 import { PluginTreeViewNodeLabelProvider } from './view/plugin-tree-view-node-label-provider';
+import { ElectronSecurityToken } from '@theia/core/lib/electron-common/electron-token';
+
+let electronRemote: typeof import('electron').remote | undefined;
```

Comment:
should all electron stuff be in a separate `src/main/electron` folder ?

Reply:
I mean, I should see no electron characters/string/imports at all in `src/browser`